File: app/src/main/java/com/vkpapps/thunder/ui/fragments/PhotoViewFragment.java
package com.vkpapps.thunder.ui.fragments;

import android.content.Context;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.appcompat.widget.AppCompatImageView;
import androidx.fragment.app.Fragment;

import com.squareup.picasso.Picasso;
import com.vkpapps.thunder.R;
import com.vkpapps.thunder.interfaces.OnNavigationVisibilityListener;

import java.io.File;

public class PhotoViewFragment extends Fragment {
    private AppCompatImageView imageView;
    private OnNavigationVisibilityListener onNavigationVisibilityListener;

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater,
                             @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_photo_view, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        imageView = view.findViewById(R.id.imageView);
        imageView.setAdjustViewBounds(true);
        Bundle arguments = getArguments();
        if (arguments != null) {
            String path = arguments.getString("PATH");
            if (path != null)
                Picasso.get().load(new File(path)).into(imageView);

        }
    }

    @Override
    public void onAttach(@NonNull Context context) {
        super.onAttach(context);
        if (context instanceof OnNavigationVisibilityListener) {
            onNavigationVisibilityListener = (OnNavigationVisibilityListener) context;
            onNavigationVisibilityListener.onNavVisibilityChange(false);
        }
    }

    @Override
    public void onDetach() {
        super.onDetach();
        onNavigationVisibilityListener = null;
    }

    @Override
    public void onDestroy() {
        super.onDestroy();
        imageView = null;
    }

}

File: app/src/main/java/com/vkpapps/thunder/ui/fragments/PhotoViewFragment.kt
package com.vkpapps.thunder.ui.fragments

import android.content.Context
import android.net.Uri
import android.os.Bundle
import android.view.*
import androidx.appcompat.widget.AppCompatImageView
import androidx.fragment.app.Fragment
import com.squareup.picasso.Picasso
import com.vkpapps.thunder.R
import com.vkpapps.thunder.interfaces.OnNavigationVisibilityListener

class PhotoViewFragment : Fragment() {
    private var imageView: AppCompatImageView? = null
    private var onNavigationVisibilityListener: OnNavigationVisibilityListener? = null
    override fun onCreateView(inflater: LayoutInflater,
                              container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_photo_view, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        setHasOptionsMenu(true)
        imageView = view.findViewById(R.id.imageView)
        imageView?.adjustViewBounds = true
        val arguments = arguments
        if (arguments != null) {
            val path = arguments.getString("Uri")
            if (path != null) Picasso.get().load(Uri.parse(path)).into(imageView)
        }
    }

    override fun onAttach(context: Context) {
        super.onAttach(context)
        if (context is OnNavigationVisibilityListener) {
            onNavigationVisibilityListener = context
            onNavigationVisibilityListener!!.onNavVisibilityChange(false)
        }
    }

    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
        super.onCreateOptionsMenu(menu, inflater)
        menu.findItem(R.id.menu_transferring)?.isVisible = false
        menu.findItem(R.id.menu_sorting)?.isVisible = false
    }


    override fun onDetach() {
        super.onDetach()
        onNavigationVisibilityListener = null
    }

    override fun onDestroy() {
        super.onDestroy()
        imageView = null
    }
}